### ISSUE TITLE
Incorrect implementation of Radio Button Grouping

### ISSUE THREAD

**Issue by avaaec** [ManifoldScholar/manifold#2320]
This is a global issue that appears in many places around the Manifold front and back end. For this issue report I will focus on a single instance.
I'm not sure if this is a new issue or one that we just missed in our first sprint of issue reporting.
## Steps to Reproduce
1. Navigate to the reader
2. Open the reader appearance submenu
3. Navigate to a radio button grouping
4. Notice that each radio button is a tab stop

## Expected / Observed 
The spec for radio button groupings can be found [here](https://www.w3.org/TR/2016/WD-wai-aria-practices-1.1-20160317/examples/radio/radio.html). Each grouping of radio buttons should be one tab stop.

## Test Environment
NVDA 2019.1.1
ESR Firefox 60.8.0esr

**Comment by zdavis** [comment]
Many thanks, @avaaec — I don't think you missed it. We recently changed these controls to radio buttons in response to other feedback your group gave us. We'll circle back and be sure to get the grouping right. Thank you very much for your feedback on this!
